src/App.js

```diff
@@ -98,12 +100,72 @@ function App() {
   function moveToHistory(input) {
     setHistoryItems([...historyItems, ...shoppingCard]);
     setShoppingCard([]);
+    window.confirm(
+      `Sehr geehrte/r ${inputData.name}! Danke für Ihren Kauf! Wir leiten Sie zu dem Kaufübersicht weiter.`
+    );
```

Comment:
It's UX wise very confusing to give the user the option to cancel here.

Reply:
done. thank you